kubernetes/crds.yaml

```diff
@@ -0,0 +1,357 @@
+# The set of CRDs used by machine-controller-manager
+#
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: machineclasses.machine.sapcloud.io
+spec:
+  conversion:
+    strategy: None
+  group: machine.sapcloud.io
+  names:
+    kind: MachineClass
+    listKind: MachineClassList
+    plural: machineclasses
+    shortNames:
+      - machcls
+    singular: machineclass
+  scope: Namespaced
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+      schema:
+        openAPIV3Schema:
+          type: object
+          x-kubernetes-preserve-unknown-fields: true
+      subresources:
+        status: {}
+
+---
+
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: awsmachineclasses.machine.sapcloud.io
+spec:
+  conversion:
+    strategy: None
+  group: machine.sapcloud.io
+  names:
+    kind: AWSMachineClass
```

Comment:
AWS?

Reply:
Yeah that was a copy paste mistake :-)

Reply:
Edit: 

In the crd file all the MachineClasses are included. The Kubevirt Machine Class can be found here: 
https://github.com/gardener/machine-controller-manager-provider-kubevirt/pull/2/files#diff-57ae387e5c536e1ade59b186d5dc2d1cR310